### PR TITLE
updated hardware info page

### PR DESCRIPTION
**Pull request by MalteKiefer**
@Ralumbi 

- Updated .gitignore
- Remove old staticfiles builds
- Updated Sidebar
- Updated Hardware Info page

image

### CODE REVIEW

File: staticfiles/core/assets/js/fastcp.js.LICENSE.4e7a22fccf2a.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*!
- * Vue.js v2.6.14
- * (c) 2014-2021 Evan You
+ * Vue.js v2.7.10
+ * (c) 2014-2022 Evan You
```
Comment (Ralumbi) [inline staticfiles/core/assets/js/fastcp.js.LICENSE.4e7a22fccf2a.txt:3]: This should stay 2021, we are taking it over from here so until that point it should stay that way and make our own notes to it.

File: staticfiles/core/assets/js/fastcp.js.LICENSE.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*!
- * Vue.js v2.6.14
- * (c) 2014-2021 Evan You
+ * Vue.js v2.7.10
+ * (c) 2014-2022 Evan You
```
Comment (Ralumbi) [inline staticfiles/core/assets/js/fastcp.js.LICENSE.txt:3]: This should stay 2021, we are taking it over from here so until that point it should stay that way and make our own notes to it.

File: staticfiles/core/assets/js/fastcp.js.LICENSE.4e7a22fccf2a.txt (hunk 1 above)
Comment (MalteKiefer) [reply staticfiles/core/assets/js/fastcp.js.LICENSE.4e7a22fccf2a.txt:3]: That is the developer from Vue, and the latest version is 2.7.10, nothing todo with FastCP

File: staticfiles/core/assets/js/fastcp.js.LICENSE.txt (hunk 2 above)
Comment (MalteKiefer) [reply staticfiles/core/assets/js/fastcp.js.LICENSE.txt:3]: That is the developer from Vue, and the latest version is 2.7.10, nothing todo with FastCP

**Pull request merged**